aten/src/ATen/native/cuda/PointwiseOpsKernel.cu

```diff
@@ -11,6 +11,12 @@
 
 namespace at::native {
 
+void addcmul_cuda_scalar_tensor2_kernel(
+  TensorIteratorBase& iter,
+  const Scalar& tensor2_scalar,
```

Comment:
haha bikeshedding, not sure if scalar2 is more concise/less confusing than tensor2_scalar....

Reply:
yea i do think scalar_tensor2 is better